pkg/live/apply-crd-task.go

```diff
@@ -89,4 +89,6 @@ func (a *ApplyCRDTask) Start(taskContext *taskrunner.TaskContext) {
 	}()
 }
 
-func (a *ApplyCRDTask) ClearTimeout() {}
+func (a *ApplyCRDTask) Cancel(_ *taskrunner.TaskContext) {}
```

Comment:
What is the purpose of these no-op functions? I expect it's due to an updated `interface`, but I am curious of the details

Reply:
The `ApplyCRDTask` implements the `Task` interface: https://github.com/kubernetes-sigs/cli-utils/blob/9877151bdf3a2b2de3955d07a915053836e2e0f5/pkg/apply/taskrunner/task.go#L26-L35